web-backend/src/main/java/com/statusneo/vms/model/Visitor.java

```diff
@@ -128,6 +149,9 @@ public String getPhoneNumber() {
 	}
 
 	public void setPhoneNumber(String phoneNumber) {
+		if (phoneNumber == null || !phoneNumber.matches(PHONE_NUMBER_REGEX)) {
```

Comment:
What happens if I pass phone no. in the ctor? Its fine for now but the right way to handle this is javax-validation using field annotations or a custom validator